adoptopenjdk-api-v3-frontend/pom.xml

```diff
@@ -14,25 +14,15 @@
     <dependencies>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-metrics</artifactId>
```

Comment:
Removed all smallrye-metrics completly as discussed with John in Slack since it's not used at all: https://adoptium.slack.com/archives/CFUJV9XV0/p1618228335054700?thread_ts=1617970411.051400&cid=CFUJV9XV0

That includes `StatsInterceptor.kt` and `DayMeter.kt` as well.